CMakeLists.txt

```diff
@@ -147,7 +157,11 @@ if(WIN32 AND NOT MSVC)
 endif()
 
 if(WIN32)
-	target_link_libraries(infoware PRIVATE gdi32 version Ole32 OleAut32 wbemuuid ntdll)
+	if(NOT MINGW)
+		target_link_libraries(infoware PRIVATE gdi32 version Ole32 OleAut32 wbemuuid ntdll)
+	else()
+		target_link_libraries(infoware PRIVATE gdi32 version ole32 oleaut32 wbemuuid ntdll)
```

Comment:
Would this not also work without the conditional by always requesting all-lowercase?

Reply:
That appears to be the case.